src/test/scala/latis/input/HapiJsonAdapterSuite.scala

```diff
@@ -2,21 +2,17 @@ package latis.input
 
 import java.net.URI
 
-import org.scalatest.flatspec.AnyFlatSpec
-import org.scalatest.matchers.should.Matchers._
-import org.scalatest.Inside.inside
+import munit.CatsEffectSuite
 
-import latis.data._
-import latis.data.DomainData
+import latis.data.{DomainData, _}
```

Comment:
Same as `import latis.data._`

Reply:
Hmm, for some reason the import optimizer doesn't see it. Must be how it slipped by.